### PR TITLE
feat: Playbook & docs contribution guides

### PR DESCRIPTION
**Pull request by he3als**
### Type
- [x] Rewrite already written documentation
- [x] Add/remove new pages

### Questions
- [x] Did you preview your changes beforehand?
- [x] Did you read the [Atlas Contribution Guidelines](https://docs.atlasos.net/contributions/)?

### Describe your pull request

- Contributing section in 'Home'
- Adds more leniency for team members on committing to primary branches without a PR
	- Only if the change is 'uncontroversial and well-tested'
- Adds to the FAQ page about testing
- Adds Playbook contribution guide
	- What Playbooks are
	- Technologies used by Atlas
	- How to build the Playbook
	- Compares VM software
	- Explains snapshotting
	- Which snapshots to make
- Adds docs contribution guide
	- What the docs are made of
	- Installing dependencies
	- Viewing changes

<https://contribution-guidelines-jeme.atlasos-docs-enq.pages.dev/contributing/contribution-guidelines/>

### CODE REVIEW

File: docs/contributing/playbook.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+|                              **Guest Tools**                              | :material-alert-circle-outline: Annoying to work with as it's [just an RDP session](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/enhanced-session-mode), low FPS |                                                                        :material-check-all: Has a vGPU, 60fps viewport, good drag-and-drop support, stable                                                                         |
+
+!!! note "Apple-silicon Mac users"
+
+    Due to not being tested frequently, we would appreciate it if you could test using Atlas with Windows-on-ARM using your Apple-silicon-based device. For this, consider using either:
```
Comment (the-P1neapple) [inline docs/contributing/playbook.md:75]: It's best to mention these apps provide the Windows 11 ARM64 installers since we don't offer them ourselves.

**Review by the-P1neapple — CHANGES_REQUESTED** [review]

**Review by the-P1neapple — APPROVED** [review]
Thanks for the edit. All good now

**Review by jacksyrm — DISMISSED** [review]

**Review by jacksyrm — APPROVED** [review]

**Review by RadNotRed — APPROVED** [review]
lgtm

**Review by whallin — APPROVED** [review]

File: docs/contributing/playbook.md [hunk 2]
```diff
@@ -69,6 +69,7 @@ Here's a comparison of our two recommendations:
...
 |                              **Guest Tools**                              | :material-alert-circle-outline: Annoying to work with as it's [just an RDP session](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/enhanced-session-mode), low FPS |                                                                        :material-check-all: Has a vGPU, 60fps viewport, good drag-and-drop support, stable                                                                         |
+|                           **Operating Systems**                           |                                                                       :material-check: Hyper-V is only avaliable on Windows                                                                        |                                                                                        :material-check-all: Avaliable on Linux and Windows                                                                                         |
```
Comment (the-P1neapple) [inline docs/contributing/playbook.md:72]: VMware available on Windows, Linux and MacOS

**Review by the-P1neapple — CHANGES_REQUESTED** [review]

File: docs/contributing/playbook.md (hunk 2 above)
Comment (he3als) [reply docs/contributing/playbook.md:72]: This is about VMWare Workstation, not Fusion. Fusion is mentioned just after the table

Comment (the-P1neapple) [reply docs/contributing/playbook.md:72]: yea i just realised, sorry

**Review by the-P1neapple — APPROVED** [review]

**Pull request merged**